lib/eventhandlers.js

```diff
@@ -328,6 +337,15 @@ const EventHandlers = {
 
     // console.log(e.inputType);
 
+    // Prevent duplicate input processing between keyEvent and inputFallBackEvent
```

Comment:
@phainv ,

Why is this part needed?
Based on your screenshots.  In safari with chinese ime the inputevent is fired first, followed by a keydown.  So I guess this check in the inputfallback isn't needed.